db/seeds/development/submissions_seed.rb

```diff
@@ -55,7 +55,89 @@
 )
 puts "Created school submission #{school_submission.id}."
 
-# student_submission = StudentSubmission.create(
+student_submission = StudentSubmission.create(
+  address_city: "#{Faker::Address.city}",
+  address_one: "#{Faker::Address.street_address}",
+  address_state: EnumConstants::STATES.sample,
+  address_two: "#{Faker::Address.street_address}",
+  address_zip: "#{Faker::Address.zip}",
+  allergies: EnumConstants::BOOLEANS.sample,
+  birthday: Faker::Date.between(33.days.ago, Date.today),
+  carpool: EnumConstants::CARPOOL.sample,
+  cell_phone: Faker::Base.numerify('###-###-####'),
+  ceremony_attendance: EnumConstants::BOOLEANS.sample,
+  ceremony_attendance_number: Faker::Name.first_name,
+  conference_id: Conference.active.first.id,
+  current_page: 3,
+  dietary_restrictions: EnumConstants::DIETARY_RESTRICTIONS.sample,
+  email: Faker::Internet.email,
+  emergency_consent: EnumConstants::BOOLEANS.sample,
+  exercise_limitations: Faker::Name.first_name,
+  first_name: Faker::Name.first_name,
+  gender: EnumConstants::GENDERS.sample,
+  guardian_one_email: Faker::Internet.email,
+  guardian_one_employer: Faker::Name.first_name,
+  guardian_one_first_name: Faker::Name.first_name,
+  guardian_one_job_title: Faker::Name.first_name,
+  guardian_one_last_name: Faker::Name.last_name,
+  guardian_one_phone_number: Faker::Base.numerify('###-###-####'),
+  guardian_one_phone_type: EnumConstants::PHONE_TYPES.sample,
+  guardian_one_relationship: EnumConstants::GUARDIAN_RELATIONSHIPS.sample,
+  guardian_two_email: Faker::Internet.email,
+  guardian_two_employer: Faker::Name.first_name,
+  guardian_two_first_name: Faker::Name.first_name,
+  guardian_two_job_title: Faker::Name.first_name,
+  guardian_two_last_name: Faker::Name.last_name,
+  guardian_two_phone_number: Faker::Base.numerify('###-###-####'),
+  guardian_two_phone_type: EnumConstants::PHONE_TYPES.sample,
+  guardian_two_relationship: EnumConstants::GUARDIAN_RELATIONSHIPS.sample,
+  health_conditions: EnumConstants::BOOLEANS.sample,
+  home_phone: Faker::Base.numerify('###-###-####'),
+  immunizations: EnumConstants::BOOLEANS.sample,
+  insurance: EnumConstants::BOOLEANS.sample,
+  insurance_address: "#{Faker::Address.street_address}",
+  insurance_address_city: "#{Faker::Address.city}",
+  insurance_address_state: EnumConstants::STATES.sample,
+  insurance_address_zip: "#{Faker::Address.zip}",
+  insurance_id: Faker::Name.first_name,
+  insurance_phone_number: Faker::Base.numerify('###-###-####'),
+  insurance_provider: Faker::Name.first_name,
+  insurance_other: Faker::Name.first_name,
+  is_primary: false, 
```

Comment:
Let's change this to `true` so we can test emails (we don't send an email to the alternate student).